lightning-background-processor/src/lib.rs

```diff
@@ -137,6 +137,52 @@ where A::Target: chain::Access, L::Target: Logger {
 	}
 }
 
+impl<P: Deref<Target = P2PGossipSync<G, A, L>>, G: Deref<Target = NetworkGraph<L>>, A: Deref, L: Deref>
+	GossipSync<P, Arc<RapidGossipSync<G, L>>, G, A, L>
```

Comment:
Probably doesn't matter all that much, but I believe plain old references could be used instead of `Arc`.